DCOS/dcos-testing.sh

```diff
@@ -284,25 +290,48 @@ test_windows_marathon_app() {
     # - Check if the exposed port is open
     # - Check if the DNS records for the task are advertised to the Windows nodes
     #
+    local AGENT_HOSTNAME=$1
+    local AGENT_ROLE=$2
+    local APP_ID="test-windows-app-$(echo $AGENT_HOSTNAME | tr . -)"
+    # Generate json file from template
+	eval "cat <<-EOF
```

Comment:
Can we add 4 spaces instead of a tab here ?

Reply:
Nope...It needs a tab for the last EOF . Else it will fail. No whitespaces allowed here unfortunatly

Reply:
Oky